tests/python/relay/opencl_texture/test_conv2d_nhwc_texture.py

```diff
@@ -313,10 +303,9 @@ def test_conv2d_inceptionv3_35_35_strides():
 
 
 @tvm.testing.requires_opencl
-def test_conv2d_resnet50_v2_nhwc_3c():
-    target = "opencl --device=adreno"
-    dtype = "float16"
-
+@tvm.testing.parametrize_targets("opencl -device=adreno")
+@pytest.mark.xfail(raises=tvm._ffi.base.TVMError)
```

Comment:
Why some of the tests are marked as xfail? I thought that all of them should be passed.

Reply:
Some test in `test_conv2d_nhwc_texture.py` marked as xfail because in CI we have nvidia opencl lib, and this tests fails in  the CI. Locally, on my nvidia device, i also have this problem, but when I tried to run tests on my adreno device, everything works fine.